styles/variables-base.less

```diff
@@ -326,9 +332,10 @@
 @agate-popup-body-margin: 0;
 @agate-popup-body-border-width: 1px;
 @agate-popup-body-border-radius: 6px;
-@agate-popup-buttons-padding: @agate-component-spacing 0 0 0;
+@agate-popup-buttons-padding: (2 * @agate-component-spacing) 0 0 0;
```

Comment:
I think this is a skin-specific value that we should leave alone here and override in the silicon variables file.